docs/_source/docs/architecture.rst

```diff
@@ -135,15 +135,222 @@ to demonstrate ways of organising different projects.
     non-production environments, then deployed to production by simply referencing a different
     var-file.
 
-    .. code-block:: yaml
-
-        - config
-            - config.yaml
-            - project
-                - vpc.yaml
-                - network.yaml
-                - database.yaml
-        - vars
-            - production.yaml
-            - staging.yaml
-            - development.yaml
+    .. code-block:: text
+
+        config/
+            config.yaml
+            project/
+                vpc.yaml
+                network.yaml
+                database.yaml
+        vars/
+            production.yaml
+            staging.yaml
+            development.yaml
+
+A complete Sceptre project example, explained
+---------------------------------------------
+
+This is a detailed example project setup for all infrastructure needs for a given application called
+"Indigo." Here are a few basic assumptions of this example:
+
+* Indigo's infrastructure will be deployed automatically using a CI/CD system (such as Jenkins).
+* The AWS role held by the CI/CD system cannot have sweeping, admin-level IAM permissions. It is
+  used by other teams in the company and cannot have its permissions modified for the Indigo
+  project.
+* Not all infrastructure in the organization is deployed/maintained by Sceptre and Indigo will need
+  to interact with that infrastructure.
+* All infrastructure for Indigo should be managed by Sceptre, including any required deployment
+  resources.
+* Developers will need to be able to stand up their own, isolated environments of Indigo that do not
+  conflict with production. These environments will have some different configurations to make them
+  cheaper for the company, but otherwise be identical to production.
+
+Project Structure
+*****************
+
+.. code-block:: text
+
+    config/
+        config.yaml             # top-level project configuration
```

Comment:
I think the docs already have a special name for this config, it's the StackGroup config correct, not project config?  IMO, keeping the naming consistent will help users.

Reply:
You're right that it's a StackGroup. But the top level StackGroup is referred to as the project (thus the "project_code" configuration key). I'll clarify this a little better, though.